packages/plugin-react-navigation/types/bugsnag-plugin-react-navigation.d.ts

```diff
@@ -0,0 +1,22 @@
+import { Plugin } from '@bugsnag/core'
+import { NavigationContainer } from '@react-navigation/native'
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface BugsnagPluginReactNavigation extends Plugin { }
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
+declare class BugsnagPluginReactNavigation {
+  constructor()
+}
```

Comment:
Is there a reason to prefer this over

```ts
declare class BugsnagPluginReactNavigation implements Plugin { ... }
```

?

Reply:
It's so that the class dynamically recieves all of the members of the `Plugin` interface. More context/discussion here https://github.com/bugsnag/bugsnag-js/pull/1067#discussion_r494207534 resulting in less duplication.

Since the duplication is minimal (just the `load` signature) I think it would be less confusing to duplicate so I'll switch.